authorization_code/app.js

```diff
@@ -7,137 +9,94 @@
  * https://developer.spotify.com/web-api/authorization-guide/#authorization_code_flow
  */
 
-var express = require('express'); // Express web server framework
-var request = require('request'); // "Request" library
-var querystring = require('querystring');
-var cookieParser = require('cookie-parser');
-
-var client_id = '03ffe0cac0a0401aa6673c3cf6d02ced'; // Your client id
-var client_secret = 'a57c43efb9644574a96d6623fb8bfbc2'; // Your client secret
-var redirect_uri = 'http://localhost:8888/callback'; // Your redirect uri
-
-/**
- * Generates a random string containing numbers and letters
- * @param  {number} length The length of the string
- * @return {string} The generated string
- */
-var generateRandomString = function(length) {
-  var text = '';
-  var possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-
-  for (var i = 0; i < length; i++) {
-    text += possible.charAt(Math.floor(Math.random() * possible.length));
-  }
-  return text;
-};
+const Spotify = require('spotify-web-api-node');
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const querystring = require('querystring');
+
+const CLIENT_ID = '03ffe0cac0a0401aa6673c3cf6d02ced';
```

Comment:
Hi @kauffecup was browsing this repo as well and noticed your PR.

I like the updates but did you mean to leave your credentials in this pull request?